README.md

```diff
@@ -27,6 +31,54 @@ It can be used in one of the following ways.
 - Use `-debug` of `--debug` option
 - Set `SD_CMD_DEBUG_LOG` environment variable to `true`
 
+### Validate
+Validating if yaml is correct as sd-cmd format.
+```bash
+USAGE:
+   sd-cmd validate [command options]
+
+OPTIONS:
+   -f, --f    Specify the path of yaml to validate (default: sd-command.yaml)
+
+EXAMPLE:
+   sd-cmd validate -f ./sd-command.yaml
+```
+
+### Publish
+Publishing a command specified by a yaml.
+```bash
+USAGE:
+   sd-cmd publish [command options]
+
+OPTIONS:
+   -f, --f    Specify the path of yaml to publish (default: sd-command.yaml)
+   -t, --t    Specify the tag given to the command (default: latest)
+
+EXAMPLE:
+   sd-cmd publish -f ./sd-command.yaml -t latest
+```
+
+### Promote
+Giving a `tag` to a `targetVersion` of command. If a `tag` is already set to another version, that tag will be moved to `targetVersion`. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).
+```bash
+USAGE:
+   sd-cmd promote namespace/name targetVersion tag
```

Comment:
Users will not be able to determine whether it is a fixed string or a variable string.
```suggestion
   sd-cmd promote namespace/name [targetVersion] [tag]
```